modules/mini-games/Race.lua

```diff
@@ -261,15 +261,20 @@ local function stop()
     end
 
     -- Print the place that each player came
+    local airtable = {}
     for name, value in pairs(scores["finish_times"]) do
         local time = value[2]
         local place = Nth(value[1])
         local colour = colors[place] or colors.default
+        if colour ~= colors.default then
+            airtable[value[1] + (value[1]-1)] = name
+            airtable[value[1] * 2] = math.round(time,2)
```

Comment:
Abit unclear to look at.
```suggestion
            local index = value[1] * 2
            airtable[index-1] = name
            airtable[index] = math.round(time,2)
```

Reply:
agreed i dont really get it myslef 😛